README.md

```diff
@@ -418,34 +517,36 @@ For `my` and `as` capabilities limited to some scheme, the action MUST be one no
 {"with": "my:dns", "can": "*"}
 ```
 
-## Proof Field Redelegation
+## 4.3 UCAN Redelegation
 
-### `prf` Scheme
+### 4.3.1 `ucan` Scheme
```

Comment:
> any reference discussion for this change ? prf:+ index or wildcard made sense to me

There was some discussion in this pr. I have suggested `ucan:` and `cid` as opposed to `prf:index` encoding.

Reply:
`prf:<index|*>` is pretty clear go look at the `prf` property at position X, `ucan:cid` duplicates the cid and its not clear that you are referring to a proof what if the proof is not a ucan ?

`ucan:cid` implies that we dont need to actually have the `prf` because we already know the cid ?